### ISSUE TITLE
missing :node-path and broken throw+ call

### ISSUE THREAD

**Issue by geoffsalmon** [tbatchelli/vmfest#19]
Is vmfest supposed to create the node-path directory automatically?  It wasn't created for me and I didn't see it in readme usage instructions. I'm using vmfest-0.2.4-beta.4, and (:node-path vmfest.manager/_location_) is "/home/geoff/.vmfest/nodes".

At first the node-path missing problem was hidden because I'm getting a "clojure.lang.Keyword cannot be cast to java.lang.String" exception from slingshot when it's trying to format the exception's message.

The throw+ call on line 294 in manager.clj needs its arguments wrapped in a map.

``` clojure
(throw+
       {:type :path-not-found
        :message (format "Path for saving nodes doesn't exist: %s" f)})
```

instead of

``` clojure
(throw+
       :type :path-not-found
       :message (format "Path for saving nodes doesn't exist: %s" f))
```

**Comment by hugoduncan** [comment]
@tbatchelli should the directory be created automatically? I think I was against this due to vmfest messing up my mountpoints when using vmfest from removable media - maybe this is a use case we just decide not to support.

**Comment by tbatchelli** [comment]
Creating the target path for the nodes when it doesn't exist should be the default behavior.

What is exactly the reason why vmfest was messing around with the mount points? Maybe there is a way to get around this issue without making it harder on first-time users...

**Comment by tbatchelli** [comment]
fixed by SHA: fac2c0b04084e2d65a6b9d9ab700f3b9757d6f3f , which will create the directory if it doesn't already exist.

**Comment by hugoduncan** [comment]
If  the path is set to /Volumes/mydisk/images/vmfest/models, say, and 'mydisk' is not mounted, then vmfest creates the path, and the disk is no longer mountable. Maybe it is sufficient to not create the path if it starts with "/Volumes/".
